packages/kubernetes-1.16/kubelet-config

```diff
@@ -84,3 +84,5 @@ tlsCipherSuites:
 - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
 maxPods: {{default 110 settings.kubernetes.max-pods}}
 staticPodPath: "/etc/kubernetes/static-pods/"
+containerLogMaxSize: {{default "10Mi" settings.kubernetes.container-log-max-size}}
```

Comment:
Thank you for submitting this PR. I think we should utilize kubelet's default instead of hard-coding what is currently kubelet's default. That way, if Kubernetes changes these values in the future, we would not inadvertently be hard-coding an old default value.

This suggestion would obviously be repeated for all of the k8s variants.

```suggestion
{{~#if settings.kubernetes.container-log-max-size  includeZero=true}}
containerLogMaxSize: {{settings.kubernetes.container-log-max-size}}
{{~/if}}
```